### ISSUE TITLE
Landing screen should showed HereNow and Featured Contents

### ISSUE THREAD

**Issue by kenle99** [ventuso/Fi-Test#54]
When tap on the merchant, and tap on the popup box, it should take to the merchant detail screen, not the list view screen.  See attached images.

**Comment by kenle99** [comment]
Here are the images.
![img_0904](https://cloud.githubusercontent.com/assets/8815191/6785484/c5aaecb0-d153-11e4-84fa-451512b1b254.PNG)
![img_0903](https://cloud.githubusercontent.com/assets/8815191/6785490/cda52886-d153-11e4-909d-e46707d209c3.PNG)

**Comment by tomtheall** [comment]
I agree with Ken. Here are 2 attachments with additional info and details on this functionality.
Notice also the detail of the back buttons behaving differently depending on where user is coming from.

Hope the explanation below is clear:

![edits-01](https://cloud.githubusercontent.com/assets/11476339/6994070/69c0321c-dad0-11e4-8bbe-57d1d6cbf9f2.jpg)
![edits-02](https://cloud.githubusercontent.com/assets/11476339/6994071/69c3055a-dad0-11e4-83b7-0f8c243efb7b.jpg)
